Workflow/dalle

```diff
@@ -127,6 +128,10 @@ function run(argv) {
     })
   ]
 
+  if (organizationId && organizationId.trim() !== "") {
```

Comment:
This whole block is inconsistent with the one above, when they’re trying to do the exact same thing.